pxtblocks/fields/field_gridpicker.ts

```diff
@@ -164,6 +142,93 @@ namespace pxtblockly {
                     goog.style.setWidth(elem, largestTextItem);
                 }
             }
+        }
+
+        /**
+         * Create a dropdown menu under the text.
+         * @private
+         */
+        public showEditor_() {
+            Blockly.WidgetDiv.show(this, this.sourceBlock_.RTL, null);
+
+            this.disposeTooltips();
+
+            let options = this.getOptions();
+            //options = [options[0]];
+
+            // Container for the menu rows
+            const tableContainer = new goog.ui.Control();
+            //const tableContainer = this.getTableContainer(options);
+            this.populateTableContainer(options, tableContainer);
+
+            // Container used to limit the height of the tableContainer, because the tableContainer uses
+            // display: table, which ignores height and maxHeight
+            const scrollContainer = new goog.ui.Control();
+
+            // Needed to correctly style borders and padding around the scrollContainer, because the padding around the
+            // scrollContainer is part of the scrollable area and will not be correctly shown at the top and bottom
+            // when scrolling
+            const paddingContainer = new goog.ui.Control();
+
+            // Record windowSize and scrollOffset before adding menu.
+            const windowSize = goog.dom.getViewportSize();
+            const scrollOffset = goog.style.getViewportPageOffset(document);
+            const xy = this.getAbsoluteXY_();
+            const borderBBox = this.getScaledBBox_();
+            const div = Blockly.WidgetDiv.DIV;
+
+            scrollContainer.addChild(tableContainer, true);
+            paddingContainer.addChild(scrollContainer, true);
+            paddingContainer.render(div);
+
+            const paddingContainerDom = paddingContainer.getElement() as HTMLElement;
+            const scrollContainerDom = scrollContainer.getElement() as HTMLElement;
+            const tableContainerDom = tableContainer.getElement() as HTMLElement;
+
+            // Search bar
+            if (this.hasSearchBar_) {
+                const searchBar = document.createElement("input");
+                searchBar.setAttribute("type", "search");
+                searchBar.setAttribute("id", "search-bar");
+                searchBar.setAttribute("class", "blocklyGridPickerSearchBar");
+                searchBar.setAttribute("placeholder", "search items");
```

Comment:
Do you think "search" as the placeholder would be more generic, in case another target uses the search box?

Also, the placeholder needs to be localized, simply use `pxt.Util.lf("search")` (with the double quotes)